src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,76 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SetTest {
+
+    Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("HashSet의 size()는 3이 반환된다.")
+    void testHashSetSize() {
+
+        // when
+        int actualSize = numbers.size();
+
+        // then
+        assertThat(actualSize).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함
테스트 코드도 유지보수해야할 코드이다.
어떻게 하면 좀 더 가독성 높고 유지보수하기 좋을까?